### PR TITLE
Fix ParticleManager::handleParticleEvent coding style

### PR DESCRIPTION
**Pull request by nerzhul**
ParticleManager::handleParticleEvent  use if a == 1 { return; } if a == 2 { return; } if a == 3 { return; }
Use a proper switch case with breaks

### CODE REVIEW

**Comment by nerzhul** [comment]
@est31 @sfan5 a review for the feature freeze ?

**Comment by nerzhul** [comment]
2 :+1:  => http://irc.minetest.ru/minetest-dev/2016-03-30#i_4569527

File: src/particles.cpp [hunk 1]
```diff
@@ -346,7 +346,7 @@ void ParticleManager::step(float dtime)
 void ParticleManager::stepSpawners (float dtime)
 {
 	MutexAutoLock lock(m_spawner_list_lock);
-	for(std::map<u32, ParticleSpawner*>::iterator i = 
+	for(std::map<u32, ParticleSpawner*>::iterator i =
```
Comment (kahrl) [inline src/particles.cpp]: missing space after for

**Comment by kahrl** [comment]
Other than that, :+1:

**Comment by est31** [comment]
e082c7766afd2be8bf55b47e261544f7191bd0ed

**Pull request closed without merging**